frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -2008,6 +2008,65 @@
             });
         };
 
+        // Edit Challenge Start and End Date
+        vm.challengeDateDialog = function(ev) {
+            vm.challengeStartDate = vm.page.start_date;
+            vm.challengeStartDate = moment(vm.challengeStartDate);
```

Comment:
@takitsuse Can we make these two lines into one? `vm.challengeStartDate = moment(vm.page.start_date);`

Reply:
I personally would like to leave it to keep the consistency with L1894-L1897.
https://github.com/Cloud-CV/EvalAI/pull/2512/files/225fc493e4d10610b8e19256e74b0b626b4c019e#diff-14a845646ed48757b5c7798b7ccdddb6L1894-L1897

Reply:
I would like to leave it still to keep the consistency with line 1894th to 1897th.

Reply:
I see... but I don't think that's for good reason so we can do this change :)